src/main/webapp/app/exercises/file-upload/assess/file-upload-assessment.component.ts

```diff
@@ -393,6 +393,18 @@ export class FileUploadAssessmentComponent implements OnInit, OnDestroy {
         // When unreferenced feedback is set, it has to be valid (score + detailed text)
         this.assessmentsAreValid = hasUnreferencedFeedback;
 
+        this.calculateTotalScore();
+
+        this.submissionService.handleFeedbackCorrectionRoundTag(this.correctionRound, this.submission);
+    }
+
+    /**
+     * Calculates the total score of the current assessment.
+     * This function originally checked whether the total score is negative
```

Comment:
Somehow I don't understand this comment. The function "originally" checked that? Looks like it is still doing it ... 🤔

Reply:
This function was formed by extracting a code snipped from the "validateAssessment()" function of file-upload-assessment-component. This exact function was already present in modeling-assessment-editor-component and since I created a new function with the exact same code of calculateTotalScore found in the  modeling-assessment-editor-component, I have copied the name of the function and it's documentation for consistency.

If the function description (documentation) below is misleading, we can update it on both of these functions.
![image](https://user-images.githubusercontent.com/67344158/154082347-b810e7f8-e52d-4df5-afce-389862552cd0.png)